code-for-canada/project-thundercat
# [Pizza Test] Creating new migration to populate pizza test

# Description

This PR is introducing the migration that populates the Pizza Test. This migration is populating the Overview (before the test starts), the background information (6 sections) and the questions (10 emails).

## Type of change

- New feature (non-breaking change which adds functionality)

## Screenshot

**Overview (before the test starts):**
![image](https://user-images.githubusercontent.com/23021242/60818525-cfb79b80-a16b-11e9-8c94-c02ad1956d92.png)

**Background Informations (EN):**
![image](https://user-images.githubusercontent.com/23021242/60818569-e8c04c80-a16b-11e9-8ff2-d70e98731a9e.png)

**Background Informations (FR):**
![image](https://user-images.githubusercontent.com/23021242/60818605-f5dd3b80-a16b-11e9-8c41-40d2f800442e.png)

**Questions/Emails (EN):**
![image](https://user-images.githubusercontent.com/23021242/60818745-448ad580-a16c-11e9-9917-9f162493fc8f.png)

**Questions/Emails (FR):**
![image](https://user-images.githubusercontent.com/23021242/60818784-54a2b500-a16c-11e9-9d87-518774f812ed.png)

**Tree-View - Organizational Structure (EN):**
![image](https://user-images.githubusercontent.com/23021242/60833908-5892ff00-a18d-11e9-9c55-2fb6e0652bf7.png)

**Tree-View - Organizational Structure (FR):**
![image](https://user-images.githubusercontent.com/23021242/60833935-66e11b00-a18d-11e9-8f29-02e819a327e8.png)

**Tree-View - Team Information (EN):**
![image](https://user-images.githubusercontent.com/23021242/60833962-7c564500-a18d-11e9-81cb-0a21310c7a4a.png)

**Tree-View - Team Information (EN):**
![image](https://user-images.githubusercontent.com/23021242/60833972-837d5300-a18d-11e9-8ff1-9cd471c8f932.png)

# Testing

Manual steps to reproduce this functionality:

1.  _docker-compose down_.
2.  _docker-compose up_.
3.  Open the following APIs:
- http://localhost/api/test-questions/?test_name=emibPizzaTest
- http://localhost/api/test-meta-data/?test_name=emibPizzaTest
4.  Make sure that you get the test's content (overview, background info and questions/emails).

# Checklist

Applicable for all code changes.
